src/cts/src/TritonCTS.tcl

```diff
@@ -103,7 +103,7 @@ proc clock_tree_synthesis { args } {
           -sink_clustering_levels -tree_buf \
           -sink_buffer_max_cap_derate -delay_buffer_derate} \
     flags {-post_cts_disable -sink_clustering_enable -balance_levels \
-           -obstruction_aware -apply_ndr -insertion_delay -use_dummy_load
+           -obstruction_aware -apply_ndr -insertion_delay -no_insertion_delay -use_dummy_load
```

Comment:
Something is off here ` -insertion_delay -no_insertion_delay` only one of these should exist and the other should be the default.  @precisionmoon please clarify

Reply:
-insertion_delay is a no op option.  It is there so existing scripts don't error out.

Reply:
Then it should be removed from the documentation and marked deprecated.  Also in the README.md -no_insertion_delay is in the options list but not the command summary which -insertion_delay is the opposite.

Reply:
Fixed in https://github.com/The-OpenROAD-Project/OpenROAD/pull/4788